docs/source/deploy_rhel7_centos7.md

```diff
@@ -1,52 +1,111 @@
-# Deploying Q to Red Hat Enterprise Linux 7 / CentOS 7 / Amazon Linux 2
+# Deploying on 7 / CentOS 7 / Amazon Linux 2
```

Comment:
This seems to be missing "Red Hat Enterprise LInux" (or there's a dangling "7" after "on").

Reply:
Change made!